src/components/ParametersBar.tsx

```diff
@@ -1,43 +1,172 @@
+import { DoubleRightOutlined, DoubleLeftOutlined } from "@ant-design/icons";
 import styled from 'styled-components';
-import { ColorPicker, Divider, Col, Space, Row } from 'antd';
+import { ColorPicker, Col, Space, Row, Card } from 'antd';
 import { useEffect, useMemo, useState } from 'react';
 import type { Color } from 'antd/es/color-picker';
 import { Color as ThreeColor } from 'three';
 import { type SimulationParams } from './Simulation';
 import { type SpaceSize } from 'antd/es/space';
 
+import ParameterButton from './ParameterComponents/ParameterButton';
+import ParameterLabel from './ParameterComponents/ParameterLabel';
+
 const Container = styled(Space)`
-  background-color: #d3d3d3;
-  color: #000000;
+  background-color: #797979;
+  color: #FFF;
+
   width: 20rem;
-  height: calc(100% - 5rem);
-  border-right: 2px solid #808080;
+  height: calc(100% - 6.0rem);
+
   font-size: 2rem;
-  min-height: 90%;
   position: absolute;
   display: flex;
 
+  text-align:left;
+
+  border-top-right-radius: 20px;
+  border-bottom-right-radius: 20px;
+
+  padding:12px;
+
   @media (max-width: 760px) {
     display: none;
   }
 `;
+
+const ButtonDiv = styled.span`
+  position:absolute;
+  left:10px;
+`;
+
 const Title = styled.span`
   font-family: 'Roboto', sans-serif;
-  font-size: 1.4rem;
+  font-size: 2.0rem;
 
   @media (max-width: 760px) {
     display: none;
   }
 `;
 
+const Category = styled(Card) `
+  font-family: 'Roboto', sans-serif;
+  background-color: #797979;
+  text-align: "left";
+  font-size: 1.0rem;
+  color: #FFFFFF;
+`;
+
+const BackButton = styled.button`
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  background-color: #D9D9D9;
+  color: #464646;
+  font-size: 16px;
+  border: none;
+  cursor: pointer;
+`
+
+function ShowHideButton(props: {
+  isVisible: boolean,
+  setVisible: (inp:boolean)=>void,
+}) {
+  const isVisible = props.isVisible
+  const setVisible = props.setVisible
+
+  return(
+    <BackButton onClick={() => setVisible(!isVisible)}>
+      {isVisible ? (<DoubleLeftOutlined />) : (<DoubleRightOutlined />)}
+      </BackButton>
+  )
+}
```

Comment:
Another issue is that, when you click the 'ShowHide' button, when the par bar is retracted, the position of this button does not correspond to the position of the par bar when it is displayed. Normally, they should be at the same level.
Hide:
<img width="415" alt="Screenshot 2023-09-09 at 18 33 37" src="https://github.com/techlauncher-mlai-edge-physics/CFlowSim/assets/44810027/aeecb2c3-04f1-476b-a755-7c55516cccc3">

Display:
<img width="465" alt="Screenshot 2023-09-09 at 18 33 32" src="https://github.com/techlauncher-mlai-edge-physics/CFlowSim/assets/44810027/e6c505a4-e7e5-4622-a3de-f10bad613a75">

Reply:
buttons should now be roughly the same height now